src/devices/bus/msx/slot/cartridge.h

```diff
@@ -96,6 +96,7 @@ class msx_cart_interface : public device_interface
 	address_space &memory_space() const;
 	address_space &io_space() const;
 	cpu_device &maincpu() const;
+	required_device<cpu_device> *required_maincpu();
```

Comment:
It’d make more sense to call this `cpu_finder` or something.

Reply:
I have renamed it to get_cpu_finder()